pep-0604.rst

```diff
@@ -74,61 +76,72 @@ Simplified Syntax
     assert str | int == Union[str,int]
     assert str | int | float == Union[str, int, float]
 
-    # Instead of
-    # typing.List[typing.Union[str, int]]
+    # Instead of typing.List[typing.Union[str, int]]
     typing.List[str | int]
 
-    # list[typing.Union[str, int]] (Currently TypeError: 'type' object has no attribute '__getitem__')
-    list[int | str]
-
     # Instead of typing.Dict[str, typing.Union[int, float]]
-    typing.Dict[str, int | float]
     dict[str, int | float]
+    list[str | int]
```

Comment:
Looks like this line belongs in the previous group, and you accidentally dropped `typing.Dict[str, int | float]` here.